### ISSUE TITLE
Missing README.md file for Crypto Hamsters example

### ISSUE THREAD

**Issue by Pegahbit** [aeternity/aepp-sophia-examples#31]

**Comment by hanssv** [comment]
The example itself is also outdated, it is written for Sophia compiler version 1.1...

**Comment by hanssv** [comment]
```ML
contract CryptoHamster =
    datatype event = NewHamster(indexed int, string, indexed int)

    record state = { hamsters : map(string, int), next_id : int }

    public stateful function init() = { hamsters = {}, next_id = 0 }

    public function createHamster(hamsterName: string) =
        createHamsterByNameDNA(hamsterName, generateDNA(hamsterName))
    
    public function getHamsterDNA(hamsterName: string) : int =
        state.hamsters[hamsterName = 0]

    private function createHamsterByNameDNA(name: string, dna: int) =
        Chain.event(NewHamster(state.next_id, name, dna))
        put(state{hamsters[name] = dna, next_id = (state.next_id + 1)})
 
    private function generateDNA(name : string) : int =
        String.sha3(name) mod 10 ^ 16
```

**Comment by mradkov** [comment]
This is addressed with #62.
